app/assets/stylesheets/application.scss

```diff
@@ -83,3 +85,25 @@ body {
   font-size: 10pt;
   margin-top: 11px;
 }
+
+.player-list {
+  margin: 20px;
+  display: inline-flex;
+}
+
+ul .player-panel {
+  margin: 6px;
+  padding: 20px 10px 20px 10px;
+  background: $steely-blue;
+  color: $gold;
+}
+
+.player-panel h5 {
+  margin-top: 20px;
+  color: $gold
+}
+
+ul .player-panel .user-pic :hover {
```

Comment:
Merge rule `ul .player-panel .user-pic :hover` with rule on line 94<br>Selector should have depth of applicability no greater than 2, but was 4